acceptance-tests/pom.xml

```diff
@@ -20,7 +20,11 @@
         <cucumber.version>1.2.5</cucumber.version>
         <spring.version>5.1.15.RELEASE</spring.version>
         <selenium.version>3.141.59</selenium.version>
-        <chromedriver.version>92.0.4515.43</chromedriver.version>
+        <!--
+        Consult https://chromedriver.chromium.org/downloads
+        to find the right ChromeDriver version to use with the current browser version,
+        -->
+        <chromedriver.version>93.0.4577.63</chromedriver.version>
```

Comment:
out of curiosity, what was this version update required for? :smile:

Reply:
ChromeDriver is used by Selenium test framework as part of automated acceptance tests, targeting headless Chrome web browser.

Versions of ChromeDriver roughly track versions of Chrome, and when the latter gets updated (especially if it's a major version number bump) ChromeDriver needs updating to corresponding as well.

In this case, Chrome available on GitHub has moved forward and the build started failing as a result.